tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_component.ng.html

```diff
@@ -66,4 +66,16 @@
     </span>
     <label>Regex</label>
   </button>
+  <button
+    mat-menu-item
+    role="menuitemradio"
+    (click)="onRegexStringEdit()"
+    class="displayRegexString"
+  >
```

Comment:
Should this be only rendered under a flag?

Reply:
yes it should be. overlooked. added